.github/scripts/build.sh

```diff
@@ -30,6 +30,7 @@ echo log_file = ${WORKSPACE_ROOT}/ccache.log >>${WORKSPACE_ROOT}/ccache.conf
 export SCCACHE_IDLE_TIMEOUT=0
 export SCCACHE_CACHE_SIZE=1G
 export RUSTC_WRAPPER=sccache
+# (Note for cargo-generate to be happy) Ensure there's a $USER environment variable set with username corresponding to the uid of the user running the container
```

Comment:
Specifically, it's needed to fill the predefined placeholders `authors` and `username`. It also tries a few other environmental variables and `git config user.name/user.email`.